helm/alfresco-identity-service/alfresco-realm.json

```diff
@@ -499,122 +506,6 @@
       "useTemplateScope": false,
       "useTemplateMappers": false
     },
-    {
-      "id": "be13165d-2792-43aa-abdb-2c6cdb627184",
-      "clientId": "alfresco",
```

Comment:
I think the `alfresco` client should remain as a default in the realm file rather than adding through values.

It should always be present when deploying the Alfresco Identity Service and it feels like it's overloading the values file which should be as easy as possible to understand and change from a developer point of view.

Reply:
That makes sense to me. I'll update the PR so that the alfresco client is back in the json and the PR is limited to adding extra clients

Reply:
Now it's limited to just supporting extra clients